styles/styles.css

```diff
@@ -936,6 +957,21 @@ main .section.beau-blue {
   background-color: var(--beaublue);
 }
 
+/* section - apple-green */
```

Comment:
Maybe we can add a generic mechanism to add `background-color` though section metadata. That way we wouldn't need so many color variants.

Reply:
I was thinking of taking hash value through word document, what do you suggest?
<img width="656" alt="image" src="https://github.com/hlxsites/sunstar/assets/119420493/dae106ce-3552-40c4-9956-8993564cb33d">

Reply:
That may be too open. We can have a predefined set of shades which author can choose form.

Reply:
did you meant, instead of hash, color name should be provided by the author (supplemented by the Document Library)?

<img width="660" alt="image" src="https://github.com/hlxsites/sunstar/assets/119420493/23ff1fad-f5c6-4372-a00b-aa02c46cb927">

Reply:
Yes. Also we may want to rename `Top 65` to something generic else author may think that they can give other values like `Top 75` etc.

Reply:
yes, removed top-65 & top-20 styles.